src/qtt/measurements/acquisition/lock_in_stimulus.py

```diff
@@ -0,0 +1,84 @@
+""" Provides the LockInStimulus"""
+from qilib.configuration_helper import InstrumentAdapterFactory
+from qilib.utils import PythonJsonStructure
+
+
+class LockInStimulus:
```

Comment:
Rename to UHFLIstimulus? And also update the filename? The class is clearly UHFLI only.

Reply:
done